### PR TITLE
Update quiz 6 file to work appropriately

### PR DESCRIPTION
**Pull request by LeoCumpli21**
## Summary

This PR closes LeoCumpli21/PyZombis#22 by:
1. Adding subtitles to each exercise as *Ejercicio #*
2. Changing each exercise's id to `q6_#`
3. Rewriting the instructions of each exercise so that they are comprehensible and the `rst` formatting doesn't mess up anything.
4. Making sure python code is PEP8 compliant. I used `flake8` for this.
5. Finally, I made a test with Playwright for ex 1 (see the attached video).

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented; locators are Pyhton code
- [x] Reviewers assigned (all peers & at least 1 mentor)

## Screenshots

https://user-images.githubusercontent.com/80920858/121464567-b633b580-c979-11eb-9396-61de5dac92d5.mp4

### CODE REVIEW

File: _sources/quiz/Quiz6.rst [hunk 1]
```diff
@@ -6,287 +6,426 @@ Quiz - 6
...
 
-   B. problema_alumnos: |br|
-    tenemos dos estudiantes ayb |br|
-    a_sorri y b_sorri indican si ayb sonríen |br|
-    tenemos problemas cuando ambos sonríen o ambos |br|
-    no están sonriendo |br|
-    devuelve True cuando hay problemas |br|
+   Desarrolle la función ``alumnos_en_problemas`` que toma dos parámetros, 
+   ``a_sonrie`` y ``b_sonrie``. a y b representan dos estudiantes.
+   ``a_sonrie`` y ``b_sonrie`` indican si a y b sonríen. Cuando ambos ambos
```
Comment (y-martinez) [inline _sources/quiz/Quiz6.rst:70]:
```suggestion
   ``a_sonrie`` y ``b_sonrie`` indican si a y b sonríen. Cuando ambos 
```

File: _sources/quiz/Quiz6.rst [hunk 2]
```diff
@@ -6,287 +6,426 @@ Quiz - 6
...
-  diff21(25) -> 8 |br|
-  dica: abs(x) devuelve el valor absoluto de x |br|
+.. activecode:: q6_5
+   :nocodelens:
 
-  ~~~~
-  def diff21(n):
-       
+   Desarrolle la función ``problemas_con_loro`` que toma dos parámetros, 
+   ``hablando`` que es ``True`` o ``False``, y ``hora`` que toma un valor
```
Comment (y-martinez) [inline _sources/quiz/Quiz6.rst:205]:
```suggestion
   ``hablando`` que puede ser ``True`` o ``False``, y ``hora`` que toma un valor 
```

File: _sources/quiz/Quiz6.rst [hunk 3]
```diff
@@ -6,287 +6,426 @@ Quiz - 6
...
-  myTests().main()
+Ejercicio 8
+-----------
 
+.. activecode:: q6_8
+   :nocodelens:
 
-.. activecode:: apaga
-  :nocodelens:
+   Desarrolle la función ``quitar_caracter`` que toma a ``s`` y ``n``
```
Comment (y-martinez) [inline _sources/quiz/Quiz6.rst:341]: En el Quiz 7 hay un ejercicio igual [Ejercicio 13](http://pyar.github.io/PyZombis/master/quiz/Quiz7.html#ejercicio-13) creo que entonces debemos quitar ese y unir el enunciado de ese con este, si estas de acuerdo

File: _sources/quiz/Quiz6.rst [hunk 4]
```diff
@@ -6,287 +6,426 @@ Quiz - 6
...
+   :nocodelens:
 
-.. activecode:: apaga
-  :nocodelens:
+   Desarrolle la función ``quitar_caracter`` que toma a ``s`` y ``n`` 
+   como parámetros. ``s`` es una cadena, y ``n`` un número entero, que 
+   representa un índice. La función debe devolver una nueva cadena sin 
+   el carácter en la posición ``n``. |br| |br| 
+   Ejemplos: |br|
+   ``quitar_caracter('gatito', 1)`` -> ``'gtito'`` |br|
```
Comment (y-martinez) [inline _sources/quiz/Quiz6.rst:346]:
```suggestion
   ``quitar_caracter("gatito", 1)`` -> ``"gtito"`` |br|
```

File: _sources/quiz/Quiz6.rst [hunk 5]
```diff
@@ -6,287 +6,426 @@ Quiz - 6
...
+   ``quitar_caracter('gatito', 4)`` -> ``'gatio'`` |br|
```
Comment (y-martinez) [inline _sources/quiz/Quiz6.rst:347]:
```suggestion
   ``quitar_caracter("gatito", 4)`` -> ``"gatio"`` |br|
```

File: _sources/quiz/Quiz6.rst [hunk 6]
```diff
@@ -6,287 +6,426 @@ Quiz - 6
...
 
+.. activecode:: q6_9
+   :nocodelens:
 
+   Desarrolle la función ``intercambiar`` que toma a ``s`` como parámetro, 
+   representando una cadena. Si ``s`` tiene un tamaño menor o igual a 1, se 
+   devuelve la misma cadena. De lo contrario, la primera y última letra de ``s`` 
+   se intercambian. |br| |br| 
+   Ejemplos: |br|
+   ``intercambiar('codigo')`` -> ``'oodigc'`` |br|
```
Comment (y-martinez) [inline _sources/quiz/Quiz6.rst:402]: Cambiarlo como en la sugerencia de arriba 😃

File: _sources/quiz/Quiz6.rst [hunk 7]
```diff
@@ -6,287 +6,426 @@ Quiz - 6
...
    :nocodelens:
 
-   A. Dormir : |br| |br|
-   dia_semana y True para días de semana |br| |br|
-    vacaciones y True en vacaciones |br| |br|
-    puedes quedarte dormido cuando sea festivo o no entre semana |br| |br|
-    devuelva Verdadero o Falso cuando se vaya a dormir o no |br|  br|
+   Desarrolle la función ``dormir`` que toma dos parámetros, 
+   ``dia_semana`` y ``dia_festivo``. Ambos parámetros toman 
+   valores booleanos, es decir ``True`` o ``False``. La función actúa
```
Comment (y-martinez) [inline _sources/quiz/Quiz6.rst:18]:
```suggestion
   valores booleanos, es decir pueden ser ``True`` o ``False``. La función actúa 
```

**Review by y-martinez — CHANGES_REQUESTED** [review]
I left some comments and suggestions 👍🏼

File: _sources/quiz/Quiz6.rst (hunk 3 above)
Comment (LeoCumpli21) [reply _sources/quiz/Quiz6.rst:341]: Me parece bien! enseguida copio el enunciado del quiz 7.

File: _sources/quiz/Quiz6.rst (hunk 6 above)
Comment (LeoCumpli21) [reply _sources/quiz/Quiz6.rst]: Hecho :)

**Review by y-martinez — APPROVED** [review]
LGTM 🚀

**Review by reingart — APPROVED** [review]
Me perdí bastante en el diff para analizar los cambios, porque se mezclan mucho los ejercicios
Voy a esperar por si hay otro mentor que pueda revisarlo mejor

**Review by NicolasSandoval — APPROVED** [review]
LGTM!

**Pull request merged**